xml/art_sle_ha_install_quick.xml

```diff
@@ -788,6 +793,98 @@ softdog                16384  1</screen>
        <keycap function="control"/><keycap>C</keycap>
       </keycombo>.
    </para>
+   </sect2>
+
+   <sect2 xml:id="sec-ha-inst-quick-test-with-cluster-script">
+    <title>Testing with the &test.script; Script</title>
+    <para>
+     The command &cmd.test.script; is a tool to run standardized tests
+     for a cluster.
+     It triggers cluster failures and verify configuration to find
+     problems. Before you use your cluster in production, it is
+     recommended to use this script to make sure everything works as expected.
+    </para>
+    <para>
+     The script supports the following checks:
+    </para>
+    <itemizedlist>
+     <listitem>
+      <formalpara>
+       <title>Environment Check <option>-e</option>/<option>--env-check</option></title>
+       <para>
+        Check if hostname is resolvable, time service, watchdog, and
+       firewall.
+       </para>
+      </formalpara>
+     </listitem>
+     <listitem>
+      <formalpara>
+       <title>Cluster State Check <option>-c</option>/<option>--cluster-check</option></title>
+       <para>
+        Check cluster service, &stonith;/fence configuration, nodes,
+        and resources.
+       </para>
+      </formalpara>
+     </listitem>
+     <listitem>
+      <formalpara>
+       <title>Split Brain Check <option>--split-brain-iptables</option></title>
+       <para>
+        Simulates split brain scenario by blocking the &corosync; port.
+        Checks wheather one node can be fenced as expected.
+       </para>
+      </formalpara>
+     </listitem>
+     <listitem>
+      <formalpara>
+       <title>Killing Test Cases <option>-kill-sbd</option>/<option>-kill-corosync</option>/<option>-kill-pacemakerd</option></title>
+       <para>
+        Kill the SBD, &corosync;, and &pace; daemons.
+        After running such a test, you can find a report in
+        <filename>/var/lib/ha-cluster-preflight-check</filename>.
+        The report includes test case description, action logging,
+        and explains possible results.
+       </para>
+      </formalpara>
+     </listitem>
+     <listitem>
+      <formalpara>
+       <title>Fence Node Check <option>--fence-node</option></title>
+       <para>Fence a specific node</para>
```

Comment:
Fence -> Fences a specific node (or even: 'Fences the specified node' if one can specify a node with the option). Best check the options in a test system.

Reply:
The command line option is: `--fence-node NODE       Fence specific node`. I changed the line to:

    Fences the specific node passed from the command line.